conftest.py

```diff
@@ -1,11 +1,10 @@
 """Configuration file for the test suite."""
-
 import os
 import sys
 
-GO_BACK_A_DIR = "/../"
-GO_INTO_SRC_DIR = "src"
+BACK_A_DIRECTORY = "/../"
+INTO_SRC_DIRECTORY = "src"
 
 # set the system path to contain the previous directory
 PREVIOUS_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
-sys.path.insert(0, PREVIOUS_DIRECTORY + GO_BACK_A_DIR + GO_INTO_SRC_DIR)
+sys.path.insert(0, PREVIOUS_DIRECTORY + BACK_A_DIRECTORY + INTO_SRC_DIRECTORY)
```

Comment:
I think the `conftest.py` should be in the `tests` directory since it's configuring the path of the test cases right? And the commit before you moved out `conftest` is also not broken

Reply:
That seems reasonable... I asserted that it broke the build because on my fork build that commit did not pass properly (because it takes forever for builds in this repository to complete). I don't understand what would be different between those two builds though.

Reply:
I see. I was gonna comment on the build too. I have seen it taking over an hour to complete a single build. Is it necessary to have four builds every commit though? It is too slow to the point that it's not really helping the development.

Reply:
I would argue no, but that was here before I started work on this branch and I believe that it was Professor Kapfhammer's decision to include python 3.6.8 as well as 3.7.3.